client/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:8.4.0
+FROM node:8.11
```

Comment:
Why are you upgrading versions of most of the components in `Dockerfile` ? Why it is required?

Reply:
I have upgraded the versions, as these versions are now mulit-arch supported and will help building scope on many platforms easily. For me it helped to build on ppc64le.

Reply:
ok!  We need to make sure that these upgrades don't break anything.